Source/Core/DiscIO/VolumeDirectory.cpp

```diff
@@ -25,7 +25,7 @@
 namespace DiscIO
 {
 static u32 ComputeNameSize(const File::FSTEntry& parent_entry);
-static std::string ASCIIToLowercase(std::string str);
+static std::string ASCIIToUppercase(std::string str);
```

Comment:
> The hardware test purpose would be to check the filename order.

Check that the filename order of what is what? Do you want us to check that the filename order on discs released by Nintendo is identical to the order we get after running Dolphin's VolumeDirectory code on the files from those discs? If so, there's no point in making this a hardware test rather than just doing it on a regular computer like I assume ds84182 already has done. If not, please tell me which two data sources the hardware test is supposed to compare.

> Windows also has equivalent functions: https://msdn.microsoft.com/en-us/library/chd90w8e.aspx

Sure, but having ifdefs for multiple platforms is really not any simpler than writing a small function that transforms to lowercase/uppercase. And I still don't know of any solution to the problem of having to set the POSIX locale.